string vision_state
string target
float64 center_x
float64 center_y
